build/packaging/linux/conf/kcnd_baobab.conf

```diff
@@ -37,6 +37,7 @@ WS_PORT=8552
 WS_ORIGINS="*"
 
 # Setting 1 is to enable options, otherwise disabled.
+AUTO_RESTART=1
```

Comment:
It seems that CNs are set to 1 and ENs are set to 0, right? 
If so, could you please explain the default value by node type at the PR description?
Also, is there any reason not to restart the EN by default?

Additionally, there seems to be no `.conf` for PNs. Could you please check?

Reply:
Right. This is enabled by default in CN only. I added the description.

I added `.conf` file of PNs. Thanks.